### DISCUSSION TITLE
Migrating from Zooz 700 Series USB to Zooz 800 Series GPIO

### DISCUSSION THREAD

**Discussion by bjh45964** [zwave-js/zwave-js-ui#3417] [Q&A] [answered]
I'm using the Zwave-JS-UI docker image on a raspberry pi as a standalone ethernet-zwave bridge for HomeAssistant (to get zwave to a detached garage). Does anyone know of a clean way to migrate from a USB Zooz controller to the GPIO based controller? I've used Simplicity Studio to do USB->USB migrations, but I don't know of a way to use the GPIO module on a windows machine.

I'm trying to migrate without needed to rebuild automations and entities.

**Comment by robertsLando** [comment]
Dunno if a backup/restore of the nvm works in this case. @AlCalzone ?

**Reply by AlCalzone** [reply, marked as answer]
Not yet: https://github.com/zwave-js/node-zwave-js/issues/5257
